docs/supervisor/development.md

```diff
@@ -23,6 +23,25 @@ When the initializing is complete you can access the Home Assistant frontend on
 
 If you need to rebuild the base of the Supervisor run the task "Build Supervisor" and then restart the "Run Supervisor" task again when that completes.
 
+#### Using Windows WSL2
+While the system is Linux and compatible with Mac, it is also possible to build and test locally on Windows using WSL2 with Ubuntu 20.04 LTS and other similar distros. This is achieved via WSL2, which makes POSIX tasks possible within Windows via system-level integration with a Linux virtual machine. 
+:::info
+You must use WSL2. WSL is not capable of running docker, and therefore is incompatibile.  If using this WSL method, you must ensure you [set up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+:::
+
+From within your WSL2-enabled Ubuntu-20.04 terminal, install Docker.io package. 
+```bash
+sudo apt-get install docker.io
```

Comment:
Don't use `docker.io`, instead use Docker CE provided by the Docker project.

Reply:
That [requires extensive setup](https://docs.docker.com/engine/install/ubuntu/), I'd like to ask why do you prefer the Docker CE package over the built-in package repository docker.io package?

Reply:
We had a lot of problems with the Docker.io packages, hence we only support and actively recommend the official Docker CE packages from Docker themselves.

<https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md#supported-operating-system-system-dependencies-and-versions>

Reply:
PS: They offer a 2 line method, not sure if that works on WSL

```bash
curl -fsSL https://get.docker.com -o get-docker.sh
sudo sh get-docker.sh
```

Reply:
Note to future self: Debian instructions  https://docs.docker.com/engine/install/debian/

Reply:
Note to future self: 
run `sudo update-alternatives --set iptables /usr/sbin/iptables-legacy`
within WSL:Debian reinstall
- Docker

Reply:
Note to future self:
Clean and Build instructions:
``` bash
docker rm $(docker ps -a -q)
docker system prune -a
```
F1: Tasks: Run Tasks-> Update Supervisor Panel
F1: Tasks: Run Tasks-> Run Supervisor